tests/fabric/test_utils.py

```diff
@@ -138,19 +139,105 @@ def test_get_helm_pod(self, mock_get_pod):
-        "msps": {"ord_MSP": {"namespace": "ord-namespace"}},
-        "orderers": {"names": ["ord0", "ord1"], "msp": "ord_MSP"},
+        "msps": {
+            "AlphaMSP": {
+                "namespace": "alpha-namespace",
+                "orderers": {
+                    "nodes":{
+                        "ord1": {},
+                        "ord2": {}
+                    }
+                }
+            },
+            "BetaMSP": {
+                "namespace": "beta-namespace"
+            }
+        }
     }
 
     def test_get_orderers(self):
-        assert (["ord0", "ord1"] == get_orderers(opts=self.OPTS))
+        assert ({"ord2", "ord1"} == get_orderers(opts=self.OPTS, msp="AlphaMSP"))
+
+    def test_get_orderers_from_msp_with_no_orderer(self):
+        assert ([] == get_orderers(opts=self.OPTS, msp="BetaMSP"))
 
 
 class TestGetPeers:
     OPTS = {
-        "msps": {"peer_MSP": {"namespace": "peer-namespace"}},
-        "peers": {"names": ["peer0", "peer1"], "msp": "ord_MSP"},
+        "msps": {
+            "BetaMSP": {
+                "namespace": "peer-namespace",
+                "peers": {
+                    "nodes": {"peer0":{}, "peer1":{}}
+                }
+            }
+        }
+    }
+
+    def test_get_peers(self):
+        assert ({"peer0", "peer1"} == get_peers(opts=self.OPTS, msp="BetaMSP"))
+
+
+class TestGetMSPS:
+    OPTS = {
+        "msps": {
+            "BetaMSP": {},
+            "AlphaMSP": {}
+            }
+        }
+
+    def test_get_peers(self):
+        assert ({"BetaMSP", "AlphaMSP"} == get_msps(opts=self.OPTS))
+
+
+class TestGetChannels:
+    OPTS = {
+        "channels": {
+            "AChannel": {},
+            "BChannle": {}
+            }
+        }
+
+    def test_get_peers(self):
+        assert ({"AChannel", "BChannle"} == get_channels(opts=self.OPTS))
+
+
+class TestGetSecretGenesis:
+    OPTS = {
+        "msps": {
+            "AlphaMSP": {
+                "namespace": "alpha-namespace",
+                "orderers": {
+                    "nodes": {
+                        "ord1": {},
+                        "ord2": {}
+                    },
+                "secret_genesis": "secret"
+                }
+            }
+        }
+    }
+
+    def test_get_peers(self):
+        assert ("secret" == get_secret_genesis(opts=self.OPTS))
+
+
+class TestGetSecretGenesis:
+    OPTS = {
+        "msps": {
+            "AlphaMSP": {
+                "namespace": "alpha-namespace",
+                "orderers": {
+                    "nodes": {
+                        "ord1": {},
+                        "ord2": {}
+                    },
+                "secret_genesis": "secret",
+                "kafka": {"name": "kafka-hlf"}
+                }
+            }
+        }
     }
 
     def test_get_peers(self):
-        assert (["peer0", "peer1"] == get_peers(opts=self.OPTS))
\ No newline at end of file
+        assert ({"name": "kafka-hlf"} == get_kafa_configs(opts=self.OPTS))
```

Comment:
Again, `kafka`, rather than `kafa`

Also, if we are getting the kafka configs, rather than the peers, we should rename the name of the test function